server/src/models/Sandwich.js

```diff
@@ -24,6 +24,22 @@ class Sandwich extends unique(Model) {
     }
   }
 
+  async averageRating() {
+    let sum = 0
+    let average = 0
+    const reviews = await this.$relatedQuery("reviews")
+    if (reviews.length !== 0) {
+      const ratingsArray = reviews.map(review => {
+        return review.starRating
+      })
+      ratingsArray.forEach(rating => {
+        sum = sum + rating
```

Comment:
prefer using `+=`

if you want a good challenge, you could also one line this with a `reduce` method, but I will leave that decision up to you